### PR TITLE
Issue 8

### PR DESCRIPTION
**Pull request by xiam**

### CODE REVIEW

File: imagick/imagick.go [hunk 1]
```diff
@@ -295,6 +295,8 @@ func (i *Image) sizeFrames(sz *imgry.Sizing) error {
 		}
 	}
 
+	i.mw = i.mw.DeconstructImages()
```
Comment (pkieltyka) [inline imagick/imagick.go]: can you do this only when its actually an animated image..? ie > 1 frame..

Comment (pkieltyka) [inline imagick/imagick.go]: I think that would make sense..?

Comment (xiam) [inline imagick/imagick.go]: Yes, good point, it has no real effect on images with only one frame. Done.

File: imagick/imagick.go [hunk 2]
```diff
@@ -295,6 +297,10 @@ func (i *Image) sizeFrames(sz *imgry.Sizing) error {
 		}
 	}
 
+	if numberImages > 1 {
+		i.mw = i.mw.DeconstructImages()
```
Comment (pkieltyka) [inline imagick/imagick.go]: this would still run on images that are meant to be flattened

Comment (xiam) [inline imagick/imagick.go]: Right, I just added `coalesceAndDeconstruct` to fix that.

**Pull request merged**